14_2_elements.js

```diff
@@ -0,0 +1,12 @@
+function byTagName(node, tagName) {
+    const elements = node.querySelectorAll(tagName);
+    return [...elements].map(el => el.nodeName.toLowerCase());
```

Comment:
You don't have to destructure `elements` in this case. You are using `.map` which returns a new array, so you can do just `elements.map(....`.